docs/DeveloperGuide.md

```diff
@@ -257,329 +257,381 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+A private 1-to-1 home tuition teacher that:
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+1. has a need to manage not more than 50 student contacts
+2. has a need to manage admin details of each student, namely, lesson fees and payment
+3. has a need to keep track of lesson details for each student
+4. teaches lessons anytime between 8am and 10pm, on any day of the week
+5. has a need to manage a busy schedule of not more than 50 lessons
+6. prefers desktop apps over other types
+7. can type fast
+8. prefers typing to mouse interactions
+9. is reasonably comfortable using CLI apps
 
```

Comment:
```suggestion
10. Has tutees belonging to Singapore’s mainstream education systems 
```

Reply:
I added this line in the draft DG on teams since I defined acad stream and acad level to be mainstream tracks in Singapore.

Reply:
added